ITEM.tsv

```diff
@@ -1767,29 +1767,29 @@ ITEM_20150317_001766	Empty_Boots
 ITEM_20150317_001767	Top 
 ITEM_20150317_001768	Stamina +5 Increase Potion 
 ITEM_20150317_001769	Right-click to use. 
-ITEM_20150317_001770	Small HP Potion 
-ITEM_20150317_001771	$Restores 192 HP. {nl}Restores HP every 2 seconds over 15 seconds. {nl}Right-click to use.{memo Cooldown of 30 seconds.}
-ITEM_20150317_001772	HP Potion 
-ITEM_20150317_001773	$Restores 320 HP. {nl}Restores HP every 2 seconds over 15 seconds. {nl}Right-click to use. Cooldown of 45 seconds.  
-ITEM_20150317_001774	Large HP Potion 
-ITEM_20150317_001775	$Restores 640 HP. {nl}Restores HP every 2 seconds over 15 seconds. {nl}Right-click to use. Cooldown of 60 seconds. 
-ITEM_20150317_001776	Small SP Potion 
-ITEM_20150317_001777	$Restores 144 SP. {nl}Restores SP every 2 seconds for 15 seconds. {nl}Right-click to use.{memo Cooldown of 30 seconds.}
-ITEM_20150317_001778	Restores 240 SP. {nl}Restores SP every 2 seconds for 15 seconds. {nl}Right-click to use. Cooldown of 45 seconds. 
-ITEM_20150317_001779	Large SP Potion 
-ITEM_20150317_001780	Restores 480 SP. {nl}Restores SP every 2 seconds for 15 seconds. {nl}Right-click to use. Cooldown of 60 seconds. 
-ITEM_20150317_001781	Stamina Pill 
-ITEM_20150317_001782	${memo the cooldown is already stated in item popup, thus removed here}Restores 15 Stamina. Right-click to use.{memo Cooldown of 30 seconds.}
-ITEM_20150317_001783	{memo X}STA Pill (Medium) 
-ITEM_20150317_001784	Restores 30 Stamina. {nl}Right-click to use. Cooldown of 45 seconds. 
-ITEM_20150317_001785	{memo X}STA Pill (Large) 
-ITEM_20150317_001786	Restores 40 Stamina. {nl}Right-click to use. Cooldown of 60 seconds. 
-ITEM_20150317_001787	Small Recovery Potion 
-ITEM_20150317_001788	Restores 244 HP and SP. {nl}Restores HP and SP every 2 seconds for 15 seconds. {nl}Right-click to use. Cooldown of 30 seconds. 
-ITEM_20150317_001789	Recovery Potion 
-ITEM_20150317_001790	Restores 373 HP and SP. {nl}Restores HP and SP every 2 seconds for 15 seconds. {nl}Right-click to use. Cooldown of 30 seconds. 
-ITEM_20150317_001791	Large Recovery Potion 
-ITEM_20150317_001792	Restores 744 HP and SP. {nl}Restores HP and SP every 2 seconds for 15 seconds. {nl}Right-click to use. Cooldown of 30 seconds. 
+ITEM_20150317_001770	$Small HP Potion 
+ITEM_20150317_001771	$Restores 192 HP. {nl}Restores HP every 2 seconds for 15 seconds. {nl}Right-click to use. 
+ITEM_20150317_001772	$HP Potion 
+ITEM_20150317_001773	$Restores 320 HP. {nl}Restores HP every 2 seconds for 15 seconds. {nl}Right-click to use. {memo Cooldown: 45 seconds}  
```

Comment:
Why are you adding a memo tag for the Cooldown description? These tags are unseen in the game and thus you are technically removing the description from the game.

Just say
> Cooldown: 45 seconds.